versioned_docs/version-8.3/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md

```diff
@@ -33,16 +35,36 @@ The following images must be available in your air-gapped environment:
   - `web-modeler-ee/modeler-webapp`
   - `web-modeler-ee/modeler-websockets`
 
+<<<<<<< HEAD
```

Comment:
FYI there's cleanup needed from the merge conflict resolution (dropping this line and its counterpart `> > > > > > > 9c4b018e` in the end of the block)

Reply:
Fixed.
I cleaned it but without saving it. Thx @maxdanilov for noticing it 🙇